modules/cluster-logging-elasticsearch-retention.adoc

```diff
@@ -5,33 +5,21 @@
 [id="cluster-logging-elasticsearch-retention_{context}"]
 = Configuring log retention time
 
-You can specify how long the default Elasticsearch log store keeps indices using a separate _retention policy_ for each of the three log sources: infrastructure logs, application logs, and audit logs. The retention policy, which you configure using the `maxAge` parameter in the Cluster Logging custom resource (CR), is considered for the Elasticsearch roll over schedule and determines when Elasticsearch deletes the rolled-over indices.
+You can configure a _retention policy_ that specifies how long the default Elasticsearch log store keeps indices for each of the three log sources: infrastructure logs, application logs, and audit logs.
 
-Elasticsearch rolls over an index, moving the current index and creating a new
-index, when an index matches any of the following conditions:
+To configure the retention policy, you set a `maxAge` parameter for each log source in the Cluster Logging custom resource (CR). The CR applies these values to the Elasticsearch rollover schedule, which determines when Elasticsearch deletes the rolled-over indices.
```

Comment:
Small nit, but I'd use "`ClusterLogging` custom resource" here to make it consistent with later in the doc.

Reply:
Thanks. Updated.